src/status_im/notifications/core.cljs

```diff
@@ -13,20 +13,21 @@
 
 (when-not platform/desktop?
 
-  (def firebase (object/get rn/react-native-firebase "default"))
-
-  ;; NOTE: Only need to explicitly request permissions on iOS.
-  (defn request-permissions []
-    (if platform/desktop?
-      (re-frame/dispatch [:notifications/request-notifications-granted {}])
-      (-> (.requestPermission (.messaging firebase))
-          (.then
-           (fn [_]
-             (log/debug "notifications-granted")
-             (re-frame/dispatch [:notifications/request-notifications-granted {}]))
-           (fn [_]
-             (log/debug "notifications-denied")
-             (re-frame/dispatch [:notifications/request-notifications-denied {}]))))))
+  (def firebase (object/get rn/react-native-firebase "default")))
+
+;; NOTE: Only need to explicitly request permissions on iOS.
+(defn request-permissions []
+  (if platform/desktop?
+    (re-frame/dispatch [:notifications/request-notifications-granted {}])
+    (-> (.requestPermission (.messaging firebase))
+        (.then
+         (fn [_]
+           (log/debug "notifications-granted")
+           (re-frame/dispatch [:notifications/request-notifications-granted {}]))
+         (fn [_]
+           (log/debug "notifications-denied")
+           (re-frame/dispatch [:notifications/request-notifications-denied {}]))))))
+(when-not platform/desktop?
```

Comment:
Can you add an empty line here to make it clear that it is not part of the function above ?

Reply:
Done.